fluxcd/tests/conftest.py

```diff
@@ -27,8 +27,26 @@ def check(instance):
 
 
 @pytest.fixture()
-def mock_metrics():
-    fixture_file = os.path.join(os.path.dirname(__file__), "fixtures", "metrics.txt")
+def mock_metrics_v1():
```

Comment:
I don't have the full picture here, but I think it would make sense to also validate the URL you hit

You could also create a function for this since the two fixtures are identical